css/index.css

```diff
@@ -87,34 +102,174 @@ p {
 
 .middle-content {
     margin-bottom: 20px;
+    margin-left: 250px;
+    margin-right: 250px;
     border-bottom: 1px dashed black;
+   
+   
 }
 
 .middle-content h2 {
     padding: 0 2%;
     margin-bottom: 0;
+ 
+    
 }
 
 .middle-content .boxes {
     display: flex;
     flex-wrap: wrap;
     justify-content: space-evenly;
+   
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes div {
     width: 12.5%;
     height: 100px;
-    background: black;
+    background: teal;
+    margin: 20px 2.5%;
+    color: white;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+   
+}
+
+.middle-content .boxes div + div {
```

Comment:
On each individual box, the parent can style all the individual boxes, the only thing that needs to be done on the boxes is to override the color.

Reply:
Ah, ok. I knew there had to be an easier way to get that to work. 
Thank you